assets/notebooks/assembly_report.qmd

```diff
@@ -233,6 +301,83 @@ if inspect:
 
 Does the smudgeplot indicate the expected ploidy?
 
+::: {.callout-note title="what is Smudgeplot and how to interpret the Smudegeplot plot" collapse="true"}
+#### **Smudgeplot Overview**
+
+**Smudgeplot** is a visualization tool that complements **GenomeScope** by estimating **ploidy** (number of chromosome sets) and revealing genome structure using **k-mer pairs** that differ by exactly one nucleotide. It helps researchers study organisms with **unknown or complex genome architectures**, such as polyploids or highly heterozygous species.
+
+---
+
+#### **How Smudgeplot Works**
+
+1. **Input Data**  
+   - Smudgeplot uses **k-mer frequency files** generated by tools like **Jellyfish** or **KMC**.  
+   - It identifies **heterozygous k-mer pairs** (two k-mers differing by one nucleotide), which likely represent alleles or paralogs.
+
+2. **Analysis**  
+   - Plots the **sum of coverages** (CovA + CovB) against the **relative coverage ratio** (CovB / (CovA + CovB)).  
+   - Groups k-mer pairs into **smudges** (clusters) that correspond to specific ploidy levels.
```

Comment:
We need an explanation how to see this in the plot, and how to use our judgment to evaluate if the given answer is probable.